### ISSUE TITLE
feat(list vew): Select the source folder on go up/back

### ISSUE THREAD

**Issue by tordex** [lgse/strata#766]
In the list view mode when I enter to any folder and then go up/back, the first list item is selected. It is very uncomfortable with keyboard navigating. I think the source folder should be selected on go up/back, like this works in the columns view.

**Comment by JoeJoeflyn** [comment]
@ogarza what is ur behavior? i forgot

**Comment by ogarza** [comment]
Right now it goes back to top of list as designed, we can discuss of it needs to be changed.

@tordex could you explain in what workflow or scenario you would want to go back to the old entry ? What happens if that entry no longer exists?

**Comment by tordex** [comment]
This useful during the navigation using the keyboard:
* Navigate over the folders with arrows
* `Enter` to enter the selected folder
* `Alt+Left` or `Alt+Up` returns back to the parent folder
* The folder I leave is selected and I can continue navigation from the same point

If it goes to the top of list, I have to "hold down Down key" to select the next folder or use mouse.
If entry no longer exists it is ok to go to the top of list. This will happen rarely, I guess.

BTW. The same issue in the Columns view. On `Alt+Left` or `Alt+Up` resets selected item to the top of column.

**Comment by ogarza** [comment]
Are you navigating like this to search for a specific file, going one
subfolder at a time?

**Comment by tordex** [comment]
Not just searching a file. Most likely I should check the folders content.

Selecting the source folder also helps with navigation with mouse inside the long list of directories. Now I have to scroll down and remember what directory I visited last. Selecting the source folder will make this folder or screen and keep eyes focus.

**Comment by l0gicgate** [comment]
This will be fixed in next release:

https://github.com/user-attachments/assets/b1f8fe2c-e3cd-4222-884c-064d8ba41aca

**Comment by tordex** [comment]
Thank you very much!
